src/Deployment.js

```diff
@@ -15,41 +16,52 @@ function Deployment(project, options) {
   this.stderr = '';
   this.success = null;
   this.project = project;
+  this.projectConfig = config.PROJECTS[project];
   this.options = options || {};
 }
 util.inherits(Deployment, EventEmitter);
 
-Deployment.prototype.python = path.resolve('venv', 'bin', 'python');
-Deployment.prototype.deployScript = path.resolve('deploy.py');
-
-Deployment.prototype.run = function() {
-  var proc = childProcess.spawn(this.python, [this.deployScript, this.project]);
-  proc.stdout.setEncoding('utf8');
-  proc.stderr.setEncoding('utf8');
-
-  proc.stdout.on('data', function(data) {
+Deployment.prototype.run = function run() {
+  var onStdout = function onStdout(data) {
```

Comment:
I am not sure if it really is necessary, but `var ... = function` creates the function on runtime not when the file is parsed, thus the function is created each time `run` is called which I thought was a good idea for eventlisteners.